lib/chef/cookbook_site_streaming_uploader.rb

```diff
@@ -36,7 +38,7 @@ class CookbookSiteStreamingUploader
     class << self
 
       def create_build_dir(cookbook)
-        tmp_cookbook_path = Tempfile.new("chef-#{cookbook.name}-build")
+        tmp_cookbook_path = Tempfile.new("#{Chef::Dist::EXEC}-#{cookbook.name}-build")
```

Comment:
I think what we need is some sort of "shortname" value. If we could do these over again they probably would be chef-client-whatever, but it's too late to change that.